src/main/java/uk/gov/hmcts/reform/waworkflowapi/duedate/RealDateService.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.waworkflowapi.duedate;
+
+import org.springframework.stereotype.Component;
+
+import java.time.ZonedDateTime;
+
+@Component
+public class RealDateService implements DateService {
+    @Override
+    public ZonedDateTime now() {
+        return ZonedDateTime.now();
+    }
+}
```

Comment:
Suggestion, maybe the above does not need to be defined explicitly and we could use lambdas like the below example: 

`public class David {

    void test() {
        DateService dateService = ZonedDateTime::now;
        System.out.println(dateService);
    }
}`

Reply:
Done